pkg/controller/certificates/informers.go

```diff
@@ -40,7 +41,7 @@ func EnqueueCertificatesForResourceUsingPredicates(log logr.Logger, queue workqu
 	return func(obj interface{}) {
 		s, ok := obj.(metav1.Object)
 		if !ok {
-			log.Info("Non-Object type resource passed to EnqueueCertificatesForSecretUsingPredicates")
+			log.V(logf.DebugLevel).Info("Non-Object type resource passed to EnqueueCertificatesForSecretUsingPredicates")
```

Comment:
This seems like it should be logged as error. It should never happen, but if it does, it should always be logged, I think.